rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -2466,17 +2440,99 @@ public void scriptCorrectlySpends_invalidScript() {
     }
 
+    private Federation getFederation(BridgeConstants bridgeConstants, String... fedKeys) {
+        // Creates retiring federation
+        List<BtcECKey> keys = Arrays.stream(fedKeys).map(s -> BtcECKey.fromPrivate(Hex.decode(s))).collect(Collectors.toList());
+        keys.sort(BtcECKey.PUBKEY_COMPARATOR);
+
+        return new Federation(
+                FederationTestUtils.getFederationMembersWithBtcKeys(keys),
+                Instant.ofEpochMilli(1000L),
+                0L,
+                bridgeConstants.getBtcParams()
+        );
+    }
+
+    private void testGetAmountSentToAddressByNetwork(BridgeConstants bridgeConstants) {
+        //Federation genesisFederation = bridgeConstants.getGenesisFederation();
+        Federation activeFederation = getFederation(bridgeConstants, "fa03", "fa04");
+        Address activeFederationAddress = activeFederation.getAddress();
+
+        Federation retiringFederation = getFederation(bridgeConstants, "fa01", "fa02");
+        Address retiringFederationAddress = retiringFederation.getAddress();
+
+        Coin valueToTransfer = Coin.COIN;
+        BtcTransaction btcTx = new BtcTransaction(bridgeConstants.getBtcParams());
+        btcTx.addOutput(valueToTransfer, activeFederationAddress);
+        btcTx.addOutput(valueToTransfer, retiringFederationAddress);
+
+        Coin totalAmountExpected = valueToTransfer.multiply(2);
+        Assert.assertEquals(totalAmountExpected, BridgeUtils.getAmountSentToAddress(btcTx, bridgeConstants.getBtcParams(), activeFederationAddress, retiringFederationAddress));
+
+        btcTx = new BtcTransaction(bridgeConstants.getBtcParams());
+        btcTx.addOutput(valueToTransfer, activeFederationAddress);
+        totalAmountExpected = Coin.COIN;
+        Assert.assertEquals(totalAmountExpected, BridgeUtils.getAmountSentToAddress(btcTx, bridgeConstants.getBtcParams(), activeFederationAddress, retiringFederationAddress));
+
+        btcTx = new BtcTransaction(bridgeConstants.getBtcParams());
+        btcTx.addOutput(valueToTransfer, activeFederationAddress);
+        totalAmountExpected = Coin.COIN;
+        Assert.assertEquals(totalAmountExpected, BridgeUtils.getAmountSentToAddress(btcTx, bridgeConstants.getBtcParams(), activeFederationAddress));
+
+        btcTx = new BtcTransaction(bridgeConstants.getBtcParams());
+        btcTx.addOutput(valueToTransfer, retiringFederationAddress);
+        totalAmountExpected = Coin.COIN;
+        Assert.assertEquals(totalAmountExpected, BridgeUtils.getAmountSentToAddress(btcTx, bridgeConstants.getBtcParams(), retiringFederationAddress));
+    }
+
+    @Test
+    public void getAmountSentToAddress_coin() {
+        testGetAmountSentToAddressByNetwork(bridgeConstantsMainnet);
+        testGetAmountSentToAddressByNetwork(bridgeConstantsRegtest);
+    }
+
+    private void testGetAmountSentToAddress_no_output_for_addressByNetwork(BridgeConstants bridgeConstants) {
```

Comment:
strange casing here... maybe stick to one style :p
```suggestion
    private void test_getAmountSentToAddress_no_output_for_address_by_network(BridgeConstants bridgeConstants) {
```